src/resources/js/components/layout/App.vue

```diff
@@ -17,7 +17,7 @@
 
             <v-spacer></v-spacer>
             <SearchBar></SearchBar>
-            <UserTooltip></UserTooltip>
+            <UserTooltip :parentRefs="$refs"></UserTooltip>
```

Comment:
you don't need to pass this attribute as you can access the app $refs attribute on your UserTooltip class with the following synthax `this.parent.$refs`

Reply:
this didn't work

Reply:
I can't even get to this.$parent.functionIMade()